source/jormungandr/jormungandr/equipments/sytral.py

```diff
@@ -29,14 +29,16 @@
 
 from __future__ import absolute_import, print_function, unicode_literals, division
 
-from jormungandr import cache, app
+from jormungandr import cache, app, new_relic
 from navitiacommon import type_pb2
 from dateutil import parser
 from jormungandr.utils import date_to_timestamp
 
 import pybreaker
 import logging
 import jmespath
+import requests as requests
+import json
```

Comment:
prefer ujson

Reply:
I've removed it as I now use the Json decoder from 'requests'